filecoin-proofs/src/api.rs

```diff
@@ -135,6 +145,48 @@ pub fn seal<T: Into<PathBuf> + AsRef<Path>>(
         None,
     )?;
 
+    let mut piece_specs = Vec::new();
```

Comment:
We should factor this code into a new function of type `(Path -> &[PaddedBytesAmount]) -> Result<Vec<PieceSpec>>`.